tests/vsock.rs

```diff
@@ -14,17 +14,29 @@
  * limitations under the License.
  */
 
+#[cfg(feature="test_vm")]
 use nix::sys::socket::VsockAddr;
+#[cfg(feature="std")]
 use nix::sys::socket::{SockAddr as NixSockAddr};
+#[cfg(feature="test_vm")]
 use rand::RngCore;
 use serde::{Serialize, Deserialize};
+#[cfg(feature="test_vm")]
 use sha2::{Digest, Sha256};
+#[cfg(feature="std")]
 use std::time::Duration;
+#[cfg(feature="std")]
 use std::thread::{self, JoinHandle};
+#[cfg(feature="std")]
 use std::io::Write;
-use vsock::{get_local_cid, Std, VsockListener, VsockStream, VMADDR_CID_HOST, VMADDR_CID_LOCAL};
+#[cfg(feature="std")]
+use vsock::{Std, VsockListener, VsockStream, VMADDR_CID_LOCAL};
+#[cfg(feature="test_vm")]
+use vsock::{get_local_cid, VMADDR_CID_HOST};
 
+#[cfg(feature="test_vm")]
 const TEST_BLOB_SIZE: usize = 1_000_000;
+#[cfg(feature="test_vm")]
 const TEST_BLOCK_SIZE: usize = 5_000;
```

Comment:
Perhaps it would be useful to group these imports like this:
```
#[cfg(feature="std")]
use {
    nix::sys::socket::{SockAddr as NixSockAddr},
    std::time::Duration,
    std::thread::{self, JoinHandle},
    std::io::Write,
    vsock::{Std, VsockListener, VsockStream, VMADDR_CID_LOCAL},
};

#[cfg(feature="test_vm")]
use {
    nix::sys::socket::VsockAddr,
    rand::RngCore;
    sha2::{Digest, Sha256};
    vsock::{get_local_cid, VMADDR_CID_HOST};
    
};
```

Reply:
Good to know that that's possible! :+1: